var/spack/repos/builtin/packages/dcmtk/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Dcmtk(CMakePackage):
+    """DCMTK is a collection of libraries and applications implementing large
+       parts of the DICOM standard."""
+
+    homepage = "https://dicom.offis.de"
+    url      = "https://github.com/DCMTK/dcmtk/archive/DCMTK-3.6.3.tar.gz"
+
+    version('3.6.3', sha256='57f4f71ee4af9114be6408ff6fcafc441c349e4c2954e17c9c22c8ce0fb065bf')
+    version('3.6.2', sha256='e9bf6e8805bbcf8a25274566541798785fd4e73bd046045ef27a0109ab520924')
+
+    variant('ssl', default=True, description="Suuport DICOM Security Enhancements one")
+    depends_on('openssl', type=('build', 'link'), when="+ssl")
```

Comment:
build link is the default dependency type, so you could omit it. (not sure if I would prefer it though)

Reply:
I personally prefer to make these choices explicit rather than implicit